### DISCUSSION TITLE
If I modify the default runc path, an error will occur. How should I configure it?

### DISCUSSION THREAD

**Discussion by imaxun** [containerd/nerdctl#1830] [Q&A] [answered]
FATA[0000] failed to create shim task: OCI runtime create failed: unable to retrieve OCI runtime error (open /run/containerd/io.containerd.runtime.v2.task/k8s.io/65c4c922b0fa19a109142e91b9b8ebe07ca3ddc3f3cee84f0a3ba5642e5e3619/log.json: no such file or directory): exec: "runc": executable file not found in $PATH: <nil>: unknown 

my path is in /opt/local/bin/crun

**Comment by AkihiroSuda** [comment, marked as answer]
Duplicate of https://github.com/containerd/containerd/discussions/7956
